features/authentication.feature

```diff
@@ -6,7 +6,7 @@ Feature: User Authentication
     Then I am presented with a login page
     When I enter my correct email
     And I enter my password
-    And I click submit
+    And I click "Log in" button
```

Comment:
It's also possible to customize your Devise view so that you _do_ have a `submit` button.